aws/templates/id/id_s3.ftl

```diff
@@ -174,7 +175,13 @@
 
     [#local id = formatOccurrenceS3Id(occurrence)]
     [#local name = formatOccurrenceBucketName(occurrence) ]
-
+    [#local publicAccessEnabled = false ]
```

Comment:
introduce a local variable as we need to check configurations in a loop to find if any is enabled